clusterctl/README.md

```diff
@@ -25,7 +24,7 @@ TBD
 1. Create a `cluster.yaml`, `machines.yaml` and `provider-components.yaml` files configured for your cluster. See the provider specific templates and generation tools at `$GOPATH/src/sigs.k8s.io/cluster-api/clusterctl/examples/<provider>`. 
 2. Create a cluster 
 ```
-clusterctl create cluster -provider [google/terrraform] -c cluster.yaml -m machines.yaml -p provider-components.yaml
+clusterctl create cluster -provider [google/terraform] -c cluster.yaml -m machines.yaml -p provider-components.yaml
```

Comment:
it should actually be vsphere, not terraform
https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/cmd/create_cluster.go#L146

Reply:
and it should --provider

Reply:
Done